### ISSUE TITLE
Ruxp badly mangles the use of spectrum. 

### ISSUE THREAD

**Issue by vamitul** [cAttte/ruxp#3]
Here is a minimal repo to replicate:
https://github.com/vamitul/swc-ruxp-issue

To use spectrum web components and their react wrappers, the theme fragments must be imported:
`import '@spectrum-web-components/theme/theme-light.js';`

**Comment by cAttte** [comment]
hi there, sorry for not seeing this earlier. i've gotten your repo working, and just to be clear: the issue is that SWC styles aren't applied to the components, right? i sadly don't have time right now to set up a vanilla plug-in in order to compare its output to that of your SWC repo, so i can't really investigate much. have you been able to fix this, or at least get a lead on why it's happening? i would appreciate any other details you could have, thank you very much!

**Comment by vamitul** [comment]
Yes, the styles are not applied. No idea why, in the end I just gave up on RUXP for now. 
Best as I can tell, the required CSS for Spectrum gets removed somehow. Sorry I can't be more of use.
